client/src/components/my-mentor-card/index.tsx

```diff
@@ -45,7 +45,7 @@ export default function MyMentorCard(props: {
     props.accessToken,
     props.mentor?.thumbnail || ""
   );
-  const thumbnailAvailable = thumbnail !== "";
+  const thumbnailAvailable = /(ftp|http|https):/.test(thumbnail);
```

Comment:
are we sure this shouldn't just be `const thumbnailAvailable = Boolean(thumbnail)`?

How can thumbnail be a non-empty string and be invalid?